tests/framework/project/test_pipeline_registry.py

```diff
@@ -50,6 +46,11 @@ def register_pipelines():
     sys.path.pop(0)
 
 
+def test_pipelines_without_configure_project_throws_exception():
+    with pytest.raises(ModuleNotFoundError, match="No module named 'dummy_package'"):
```

Comment:
Hmm I thought `pipelines` should be an empty dictionary without throwing an error when the project is not configured, i.e. when settings have been loaded. I'm having trouble following but do you know why this wasn't raising before?

Reply:
It's a weird one, and I actually need to spend some more time investigating what's going on. But in short this test passed without changes if only this test file is run, but when run as part of `make test` it fails because of the setup in https://github.com/kedro-org/kedro/blob/main/tests/framework/cli/conftest.py. Several fixtures in that file have `scope=module`. When I change it to be a smaller scope this test passes again, but lots of others fail.

Reply:
Thanks for checking @AntonyMilneQB !